support-frontend/assets/helpers/page/analyticsAndConsent.ts

```diff
@@ -22,6 +25,7 @@ function analyticsInitialisation(
 	participations: Participations,
 	acquisitionData: ReferrerAcquisitionData,
 ): void {
+	sendEventAcquisitionDataFromQueryParamEvent(acquisitionData);
```

Comment:
For consistency I'd sugges calling `sendEventAcquisitionDataFromQueryParamEvent` from inside the `init` function of the `quantumMetric` module instead of here, this would follow the pattern used by `sendEventABTestParticipations`.